src/app/shared/shared.module.ts

```diff
@@ -14,10 +16,12 @@ import {HomeRoutingModule} from "../hero/home/home-routing.module";
     CardComponent,
     DetailContainerComponent
   ],
-    imports: [
-        CommonModule,
-        HomeRoutingModule
-    ],
+  imports: [
+    CommonModule,
+    HomeRoutingModule,
+    HttpClientModule,
+    NgxUiLoaderModule,
```

Comment:
is this a built in loader module? can you try using a gif instead? and try to implement a page loader etc?

Reply:
I tried to implement it, but removed it later. I will try this again for both popular Page and detail Page.

Reply:
ok you can remove it for now